pkg/mimir/modules_test.go

```diff
@@ -189,6 +190,14 @@ func TestMultiKVSetup(t *testing.T) {
 		Ingester: func(t *testing.T, c Config) {
 			require.NotNil(t, c.Ingester.IngesterRing.KVStore.Multi.ConfigProvider)
 		},
+
```

Comment:
Should the compactor also be added here?

Reply:
Yes, let's do that.

Reply:
Added in https://github.com/grafana/mimir/pull/1665/commits/be465a2b4d7dff6be2e9ff76e5d8c98f45400f74.